### PR TITLE
Light refactor and remove calls to ReactDOM

### PR DESCRIPTION
**Pull request by nhunzaker**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,25 +23,21 @@
     "babel": "~5",
     "babel-core": "~5",
     "babel-loader": "~5",
-    "babel-runtime": "~5",
-    "coveralls": "~2",
```
Comment (nhunzaker) [inline package.json:27]: I still have not figured out automated code coverage on Circle. Something todo...

**Comment by mzlock** [comment]
Looks good, from what I understand. :+1:

File: src/focal-point.jsx [hunk 2]
```diff
@@ -26,11 +25,11 @@ let FocalPoint = React.createClass({
   },
 
   contains(element) {
-    return DOM.findDOMNode(this).contains(element)
+    return this.refs.root.contains(element)
```
Comment (benjtinsley) [inline src/focal-point.jsx:28]: ah this is what you were talking about earlier

**Comment by benjtinsley** [comment]
:+1:

**Pull request merged**